tibiawikisql/__main__.py

```diff
@@ -301,23 +301,24 @@ def generate_loot_statistics(conn: sqlite3.Connection, data_store):
                 if creature_id is None:
                     # This could happen if a creature's article was deleted but its Loot Statistics weren't
                     continue
-                # Most loot statistics contain stats for older versions too, we onl care about the latest version.
+                # Most loot statistics contain stats for older versions too, we only care about the latest version.
                 kills, loot_stats = parse_loot_statistics(article.content)
```

Comment:
I think in this case it would be better to figure out why parse_loot_statistics is returning an empty entry 🤔

Reply:
I made it return an empty entry with the change on tibiawikisql/utils.py.

Since line 308 tries to add an entry["item"], the entry must have the "item" key.